### PR TITLE
Get number of documents and document parts

### PR DESCRIPTION
**Pull request by pwoznic**
Updated the docs and API reference about returning the number of docs and doc parts in a corpus. This can help with tracking usage and monitoring corpora.

### CODE REVIEW

**Comment by pwoznic** [comment]
Fixes #307 

I went through the long-form docs and added the minor changes to this PR since it will be merged early next week.

File: www/docs/api-reference/api-keys/list-api-keys.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ status and help you manage the lifecycle and security of your API keys.
 
 To list API keys, send a GET request to `/v2/api_keys`.
 
-The response includes a array of `ApiKey` objects that show the API keys and
+The response includes a array of `api_keys` objects that show the API keys and
```
Comment (tallatshafaat) [inline www/docs/api-reference/api-keys/list-api-keys.md:21]: Perhaps: The response includes an `api_keys` array field that contains information about the API keys, and a `metadata` field containing information such as pagination key.

File: www/docs/api-reference/app-clients/create-app-client.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ following properties:
 - `api_roles` - Specifies the API roles associated with the client credentials
   as `owner`, `administrator`, `billing_administrator`, or `corpus_administrator`.
 
-The response includes the `AppClient` object that contains the assigned App
+The response includes the `app_clients` object that contains the assigned App
```
Comment (tallatshafaat) [inline www/docs/api-reference/app-clients/create-app-client.md:31]: It seems like you're getting rid of schema object names like `AppClient`. If that is the case, then `app_clients` isn't the right thing to use either. It should be something like you did in the get-api-key-md page where you completely got rid of the `ApiKey` term (and didn't replace with with `api_key`.

File: www/docs/api-reference/app-clients/list-app-clients.md [hunk 3]
```diff
@@ -28,7 +28,7 @@ following properties:
 - `api_roles` - Specifies the API roles associated with the client credentials
   as `owner`, `administrator`, `billing_administrator`, or `corpus_administrator`.
 
-The response includes the `AppClient` object that contains the assigned App
+The response includes the `app_clients` object that contains the assigned App
```
Comment (tallatshafaat) [inline www/docs/api-reference/app-clients/list-app-clients.md:31]: Same.

File: www/docs/api-reference/admin-apis/corpus/read-corpus.md [hunk 4]
```diff
@@ -31,7 +31,29 @@ To get corpus metadata, send a GET request to `/v2/corpora/:corpus_key`, where
...
+usage stay within allocated quotas and identify trends in document growth and 
+document segmentation.
+
+The `limit` object in the response provides comprehensive information about the 
+current usage and limits of a corpus:
+
+* `used_docs` - The number of documents currently stored in the corpus.
+* `used_parts` - The number of document parts contained in the corpus, 
+  which is useful in cases where documents are segmented into smaller parts.
+* `used_bytes` - The total storage used by the corpus in bytes, including document
```
[the remark this answers is not in the document]
Comment (pwoznic) [reply www/docs/api-reference/admin-apis/corpus/read-corpus.md:49]: FWIW, when I regenerated the help from the master yaml, it had these fields:

image

[the remark this answers is not in the document]
Comment (tallatshafaat) [reply www/docs/api-reference/admin-apis/corpus/read-corpus.md:49]: Thanks for pointing out. let me send a PR to remove these from the yaml file too.

**Review by tallatshafaat — APPROVED** [review]

**Pull request merged**